crates/rooch-framework/src/natives/rooch_framework/crypto/encoding.rs

```diff
@@ -18,10 +18,10 @@ use move_vm_types::{
 use smallvec::smallvec;
 use std::collections::VecDeque;
 
-pub const INVALID_PUBKEY: u64 = 1;
-pub const EXCESSIVE_SCRIPT_SIZE: u64 = 2;
-pub const INVALID_DATA: u64 = 3;
-pub const INVALID_SCRIPT_VERSION: u64 = 4;
+pub const E_INVALID_PUBKEY: u64 = 1;
+pub const E_EXCESSIVE_SCRIPT_SIZE: u64 = 2;
+pub const E_INVALID_DATA: u64 = 3;
+pub const E_INVALID_SCRIPT_VERSION: u64 = 4;
```

Comment:
The error code should also be defined in the Move module, and then the `move explain` command can explain the error code.